plugin-server/src/utils/token_bucket.ts

```diff
@@ -0,0 +1,75 @@
+import { defaultConfig } from './../config/config'
```

Comment:
I _think_ the naming of javascript/typescript files uses `-` rather than `_`, at least this is how other files are configured.